flocker/node/agents/blockdevice.py

```diff
@@ -1121,8 +1121,10 @@ def list_live_nodes():
         This is used to figure out which nodes are dead, so that other
         nodes can do the detach.
 
-        :returns: A collection of ``unicode`` compute instance IDs, compatible
-            with those returned by ``IBlockDeviceAPI.compute_instance_id``.
+        :returns: A mapping of ``unicode`` compute instance IDs
+            (compatible with those returned by
+            ``IBlockDeviceAPI.compute_instance_id``) to IPs of those
```

Comment:
I misunderstood this until I read the code further down, thinking it was one IP per compute instance. perhaps `as lists of unicode values` instead of `also as unicode`?